soh/src/code/z_kankyo.c

```diff
@@ -2290,27 +2290,22 @@ Color_RGB8 sSandstormEnvColors[] = {
     { 50, 40, 0 },
 };
 
-Gfx* gFieldSandstormDL_Custom = NULL;
+u16 hasPatchedSandstormRect = 0;
```

Comment:
the name here implies a bool to me, not sure what a better one would be (maybe just `patchedSandstormRect`)

Reply:
Good call, maybe previouslyPatchedSandstormRect? A bit mouthy, I might go with yours

Reply:
it stores the last patched screen size, so that we’re only re-patching if the screen size changes (the rect drawn depends on screen size)

Reply:
So maybe patchedSandstormScreenSize

Reply:
Done 👍